lms/templates/ccx/underscore/unit.underscore

```diff
@@ -0,0 +1,58 @@
+<td class="unit">
+  <% if (unit.category !== "vertical") { %>
+    <button class="toggle-collapse ccx-button-link" aria-expanded="false">
+      <i class="fa fa-caret-right"></i>
+      <span class="sr">
+        <%- interpolate(gettext('toggle chapter %(displayName)s'),
+         {displayName: unit.display_name}, true) %>
+      </span>
+    </button>
+  <% } %>
+  <span class="sr">
+     <% if (unit.category === "chapter") { %>
+       <%- gettext('section') %>
+     <% } else if (unit.category === "sequential") { %>
+       <%- gettext('sub section') %>
+      <% } else { %>
+        <%- gettext('unit') %>
+      <% } %>
+      &nbsp;</span><%= unit.display_name %>
+  </td>
+
+  <% if (unit.category !== "vertical") { %>
+    <td class="date start-date" data-date="<%= unit.start %>" data-location="<%= location %>"></td>
+  <% } else { %>
+    <td class="no-link">
+      <% if (unit.start) { %>
+        <%= unit.start %>
+      <% } else { %>
+        <%
+        // Translators: Unit's aka vertical start date is set to Unscheduled when user has not set start date on corresponding subsection aka sequential.
+        %>
+        <%- gettext('Unscheduled') %>
+      <% } %>
+    </td>
+  <% } %>
+  <% if (unit.category === "chapter") { %>
+    <td class="no-link">
+      <%- gettext('N/A') %>
```

Comment:
This could use a translation comment. Or perhaps just change to "Not Applicable"?